portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,60 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+html{
+    height: 100%;
+    width: 100%;
 }
+ 
+body{
+    height: 100%;
```

Comment:
I suspect that this isn't needed, because the body will inherit the height from the html tag above! Although it's hard for me to really tell without seeing the page, so if I'm incorrect feel free to just keep it as-is.

Reply:
You are right!